spec/factories/courses.rb

```diff
@@ -210,12 +210,12 @@
       to_create { |instance| instance.save(validate: false) }
     end
 
-    trait :unpublished_with_primary_maths do
+    trait :unpublished do
       transient do
-        identifier { "unpublished_with_primary_maths" }
+        identifier { "unpublished" }
       end
 
-      name { "#{identifier} course name" }
+      name { "#{identifier}_course" }
```

Comment:
This is what I was talking about before, btw, about using the string "name" in here so that we can trace how this value is used through the system. I think what was here before is right:

```suggestion
      name { "#{identifier} course name" }
```

Reply:
👍